src/lib/components/Link/Link.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from "@emotion/styled";
+import { AnchorHTMLAttributes } from "react";
+
+export interface LinkProps extends AnchorHTMLAttributes<HTMLAnchorElement> {}
+
+const StyledLink = styled.a<LinkProps>`
+	font-size: 14px;
+	font-weight: 700;
+	color: ${({ theme }) => theme.colors.primary.main};
+
+	cursor: default;
+
+	border-radius: 4px;
+
+	padding: 4px 4px;
+
+	&:hover {
+		background-color: ${({ theme }) => theme.colors.background.light[500]};
```

Comment:
Let's change the text color to a lighter on hover, and apply this style on active / focus. Also, let's remove the outline on focus.